strawberryfields/backends/states.py

```diff
@@ -1250,6 +1252,19 @@ def number_expectation(self, modes):
             "The number_expectation method only supports one or two modes for Gaussian states."
```

Comment:
This only should be removed.

Reply:
I didn't add this, it was already there. Do you still want it removed?

Reply:
I didn't add this, do you still want it removed?

Reply:
I got confused. This line should stay.